node-runtime/src/context.ts

```diff
@@ -30,6 +30,12 @@ export interface ContextReply {
   result?: unknown;
 }
 
+export interface ContextResponse {
+  statusCode?: number;
+  headers?: Record<string, string>;
```

Comment:
Sugiro headers já ser inicializado para um objeto vazio para tornar mais fácil "adicionar um header" em um middleware. Isso impacta a linha 767 do http-server.ts tbm.

```suggestion
  headers: Record<string, string>;
```